src/chrome/content/rules/iTerm2.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+	For iterm2.com
+
+		Works:
+
+			- www
+			- $
+
+		Doesn't Work:
+			nil
+
+		www redirects to $
+		doing rewrite here for performance.
+-->
+<ruleset name="iTerm2">
+  <target host="iterm2.com" />
+  <target host="*.iterm2.com" />
+
+  <rule from="^http://(?:www\.)?iterm2\.com/"
+          to="https://iterm2.com/" />
+
+  <test url="http://iterm2.com/" />
```

Comment:
This test URL is unnecessary, since the ruleset checker creates an implicit URL for each target host.